compiler_opt/rl/registry.py

```diff
@@ -33,6 +33,7 @@
 # to trigger gin registration.
 import compiler_opt.rl.inlining  # pylint: disable=unused-import
 import compiler_opt.rl.regalloc  # pylint: disable=unused-import
+import compiler_opt.rl.unroll  # pylint: disable=unused-import
```

Comment:
nit: not sure whether unroll is ambiguous in meaning; if it is, probably just use loop_unroll